.golangci.yml

```diff
@@ -71,4 +71,8 @@ severity:
     - linters:
         - staticcheck
       text: "SA1019:"
-      severity: info
\ No newline at end of file
+      severity: info
+issues:
+  exclude-rules:
+    - path: _test\.go
+      text: "Potential hardcoded credentials"
```

Comment:
for some reason now it complains about some of the sha256 digests that we are using during testing